templates/templates/_navigation-enterprise-h.html

```diff
@@ -144,9 +144,9 @@ <h4 class="p-muted-heading"><a href="/server">Data centre&nbsp;&rsaquo;</a></h4>
         <h4 class="p-muted-heading"><a href="/kubeflow">AI &amp; ML&nbsp;&rsaquo;</a></h4>
         <ul class="p-text-list--small is-bordered">
           <li class="p-list__item"><a href="/kubernetes/features">Kubernetes support for AI/ML</a></li>
-          <li class="p-list__item"><a href="/blog/ai-ml-model-ubuntu">Deploy your first model with Kubeflow</a></li>
+          <li class="p-list__item"><a href="/kubeflow/install">Deploy your first model with Kubeflow</a></li>
```

Comment:
This page doesn't result in a deployed model but is a much better start to kubeflow. Maybe replace the text to something like "Start using kubeflow with MicroK8s".

Reply:
ok